Dockerfile_Distributed

```diff
@@ -0,0 +1,36 @@
+FROM python:3.10-slim-bookworm as builder
```

Comment:
todo: move this to a hazard-safe folder, its just an interim placeholder while distributed is a WIP